### DISCUSSION TITLE
ZUGFeRD and Factur-X

### DISCUSSION THREAD

**Discussion by joeken** [kimai/kimai#2364] [Questions & Answers]
Dear All,

are there any activities planned/scheduled to support ZUGFeRD and/or Factur-X compliant invoices?
Can you estimate the efforts? I assume, that solutions may be possible by means of pdf, pdf.twig or xml.

Thanks Stephan

**Comment by kevinpapst** [comment]
You can vote here: https://github.com/kevinpapst/kimai2/issues/1396

I cannot estimate efforts without having a deeper look. This topic is rather special and will likely only be added if someone pays for the integration. You can contact me via email if you want to support Kimai.
